### DISCUSSION TITLE
Filter list causing flow to freeze

### DISCUSSION THREAD

**Discussion by farkam135** [postmanlabs/postman-flows#394] [Q&A] [answered]
I am trying to do a simple task with postman flows and I am scratching my head on why it's not working. I created this sample flow which shows what I am trying to do, basically I have a list of strings and I just need to filter the list only keeping elements that start with test_. However when I run it it just keeps running and nothing gets outputted to the terminal. Any help is much appreciated!

![Image](https://i.imgur.com/rb3TtkK.png)

**Comment by shrey-bhagat** [comment, marked as answer]
Hi @farkam135 . `Start With` receives inputs from both ports. The `Start With` in your flow receives 3 string inputs from the list from one port and only one string input from another port. Hence the flow just keeps running.

One way to do this is : 
<img width="1039" alt="Screenshot 2022-09-14 at 2 20 52 AM" src="https://user-images.githubusercontent.com/98767117/190006637-eed4c806-997e-45c7-b2ae-446ab1988371.png">
